### PR TITLE
Add check if new services exist

### PR DESCRIPTION
**Pull request by aethyrxeo**
As there are no commits with changes for services to c-a-d yet, but soon we will be using new services in CI, we need to add a check to verify what services are present (new or old) and delete them.

### CODE REVIEW

**Comment by m-kostrzewa** [comment]
could you please update diagnostic check as well?

File: Clear-ComputeNode.ps1 [hunk 1]
```diff
@@ -65,14 +65,34 @@ function Remove-NodeMgrService {
...
 
 function Remove-DockerDriverService {
-    Write-Host "Stopping Docker Driver and removing service..."
-    Remove-Service -ServiceName "contrail-docker-driver"
+    $ServiceName = Get-ProperCNMPluginName
+    Write-Host "Stopping $ServiceName and removing service..."
+    Remove-Service -ServiceName $ServiceName
     Invoke-ScriptBlockAndPrintExceptions {
         # Docker Driver may run as a service. Or not.
         Stop-ProcessIfExists -ProcessName "contrail-windows-docker-driver"
```
Comment (jacek-izykowski) [inline Clear-ComputeNode.ps1:98]: Is this name still valid?

**Review by jacek-izykowski — COMMENTED** [review]
JCh & JI

File: Clear-ComputeNode.ps1 (hunk 1 above)
Comment (aethyrxeo) [reply Clear-ComputeNode.ps1:98]: Yes, there were no changes to the executable name yet.

**Review by jacek-izykowski — APPROVED** [review]

**Pull request merged**